Makefile

```diff
@@ -22,9 +22,12 @@ init: create-switch install ## Configure everything to develop this repository i
 install: ## Install development dependencies
 	yarn install
 	opam update
-	opam install -y . --deps-only --with-test
+	opam install --locked -y . --deps-only --with-test
 	opam exec opam-check-npm-deps
 
+lock:
+	opam lock .
```

Comment:
Can we avoid using lock files please? `reshowcase` is published in [the main opam repo](https://opam.ocaml.org/packages/reshowcase/), and the "opam way" is to make all packages work with the most recent versions available in the opam repository (in that sense, it works more than homebrew than like npm).

If this is not true, and some version doesn't work with a specific combination of packages, the opam ci will tell us, and we can constrain them in the main opam file after the fact.